ppo/replay_buffers/base_replay_buffer.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+import jax.numpy as jnp
+import chex
+
+from ppo.replay_buffers.transition import Transition
+
+
+class BaseReplayBuffer:
+    """Fixed-size base buffer to store transition tuples."""
+
+    def __init__(self, buffer_capacity: int) -> None:
+        self._memory = list()
+        self._maxlen = buffer_capacity
+
+    def sample_a_transition(self) -> Transition:
+        """Randomly sample a transition from memory."""
+        assert len(self._memory) > 0, "replay buffer is unfilled"
+
+        transition_idx = np.random.randint(0, len(self._memory))
+        transition = self._memory.pop(transition_idx)
+
+        return transition
+
+    def sample_full_buffer(self) -> chex.Array:
+        """Randomly sample a transition from memory."""
```

Comment:
Maybe we should change this docstring; it's the same as for the `sample_a_transition` method but they do different things.